src/main/java/seedu/duke/logic/commands/favcommand/AddFavCommand.java

```diff
@@ -12,38 +12,54 @@
 
 public class AddFavCommand extends Command {
 
-
-    private final String input;
+    /** Command to be saved by the user in the FavList.*/
```

Comment:
do self explanatory code like this require java docs?

Reply:
Up to you, I think. I only added it because I saw it on the module website
![image](https://user-images.githubusercontent.com/60455558/98443617-59ec8700-2147-11eb-8dd8-8922aee1a5c3.png)